README.md

```diff
@@ -37,4 +37,4 @@ register('/service-worker.js', {
 
 The `ready`, `registered`, `cached`, `updatefound` and `updated` events passes a [ServiceWorkerRegistration](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerRegistration) instance in their arguments.
 
-The `registrationOptions` object will be passed as the second argument to [`ServiceWorkerContainer.register()`](https://developer.mozilla.org/de/docs/Web/API/ServiceWorkerContainer/register#Parameter)
+The `registrationOptions` object will be passed as the second argument to [`ServiceWorkerContainer.register()`](https://developer.mozilla.org/en-US/docs/Web/API/ServiceWorkerContainer/register#Parameter)
```

Comment:
Thanks for spotting this!
But in the English version there should be a trailing `s` in the anchor id, could you fix it please?

Reply:
Done